proxy/src/main/java/com/wavefront/agent/queueing/TaskQueueFactoryImpl.java

```diff
@@ -110,15 +113,36 @@ public Long value() {
     try {
       File lockFile = new File(lockFileName);
       if (lockFile.exists()) {
-        Preconditions.checkArgument(true, lockFile.delete());
+        if (!Files.deleteIfExists(lockFile.toPath())) {
+          // no file with name found
+          throw new LogFilePathException();
```

Comment:
We don't care if the file can't be deleted if not found, that means that there is no other proxy blocking the buffer, we only care if the file exists and can't be deleted, and that throw a exception. So can remove the `if`.